docs/competition/team-support/tinker-time.md

```diff
@@ -1,13 +1,23 @@
 # Tinker Time
 
-Tinker Time is time allocated in the schedule to allow teams to test their robot in the arena outside of a match.
+Tinker Time is time allocated in the schedule to allow teams to test their robot in the actual arena.
 
-During Tinker Time up to 3 members of a team may enter the arena.
+Each 'slot' will last 10 minutes, 4 teams can be in the arena at once (one in each corner), robots must enter the arena through staging. Teams should be considerate of other teams and robots in the arena, using the space around the arena to watch rather than all trying to be in the arena at once. Shepherds will not be used to fetch teams for tinker time slot - they will need to remember.
```

Comment:
```suggestion
Each 'slot' will last 8 minutes, 4 teams can be in the arena at once (one in each corner), robots must enter the arena through staging. Teams should be considerate of other teams and robots in the arena, using the space around the arena to watch rather than all trying to be in the arena at once. Shepherds will not be used to fetch teams for tinker time slot - they will need to remember.
```
Not sure how much it matters for this, but tinker time is an hour each day this year, divided by the 30 (sort of) teams gives 8 minutes per team. Obviously with more drop outs it'll get closer to 10 minutes though

Reply:
Think the total should be 3hr10min not 3.5 hours. I think the reason for 8 minutes was to ensure that every team got a slot in each Tinker time period

Reply:
A good point. I think it depends so much on how much time is in the schedule for tinker time. I think times should be fairly round, but being too strict is probably unnecessary. I've added "usually" to hopefully make it clear it's more of a guideline.